Scripts/Python/xOptionsMenu.py

```diff
@@ -1403,7 +1433,12 @@ def OnTimer(self,id):
             if PtIsDemoMode():
                 gLiveMovie = ptMoviePlayer(kDemoMovieName,self.key)
             else:
-                gLiveMovie = ptMoviePlayer(kLiveMovieName,self.key)
+                gLiveMovieList = []
+                for file in os.listdir(kLiveMovieDir):
+                    if os.stat(f'{kLiveMovieDir}\{file}').st_size > 1000:
+                        gLiveMovieList.append(file)
+                gCurrentMovieName = gLiveMovieList[0]
```

Comment:
This will fail if there are no video files present.

Reply:
Line 1406 will cause it to error if no video files are present and prevent this code from being called

Reply:
Ah, so it does.  The code folding in the diff helped me miss it.

Reply:
Actually, I'm not so certain.  I believe the code at 1406 will fail to guard against this if there are files present but they are all below the size threshold.  It's also possible now for the movie list to contain files that aren't movies if they exist in this folder.